common4j/src/main/com/microsoft/identity/common/java/platform/JweResponse.java

```diff
@@ -69,59 +74,76 @@ public static class JweHeader {
 
     JweHeader mJweHeader;
 
-    String mEncryptedKey;
+    private String mEncryptedKey;
```

Comment:
Can we list what is nullable, and what is non-null?

Alternatively, I would suggest branching this class out into V2 and V3. It's probably easier to detect that way if we're getting NOT a value that is expected in V3 response.

Reply:
Annotated the fields and added comment for nullable. I think the class is meant for JWE parsing and help layers above (PRTv2 and PRTv3) read the values. The calling flow would validate if particular field were required and is correct for them.